inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/repository/DataProxyConfigRepository.java

```diff
@@ -385,7 +385,8 @@ private Map<String, List<InLongIdObject>> parseInlongId(Map<String, InlongGroupI
             InlongStreamId streamIdObj = entry.getValue();
             String groupId = streamIdObj.getInlongGroupId();
             InlongGroupId groupIdObj = groupIdMap.get(groupId);
-            if (groupId == null) {
+            if (groupId == null || groupIdObj == null) {
+                LOGGER.warn("groupId {} or groupIdObj {} is null, ignored", groupId, groupIdObj);
```

Comment:
It is better to set logging level to **_debug_**.

Reply:
OK